src/Extractor/BaseExtractor.php

```diff
@@ -41,7 +41,7 @@ public function __construct(array $parameters, array $state, LoggerInterface $lo
         $this->state = $state;
         $this->logger = $logger;
         $this->parameters = $this->createSshTunnel($this->parameters);
-        $this->databaseConfig = $this->createDatabaseConfig($parameters['db']);
+        $this->databaseConfig = $this->createDatabaseConfig($this->parameters['db']);
```

Comment:
Preklep v kode, ma sa pouzit `$this->parameters`, ktore su modifikovane vytvorenim SSH tunela (riadok `43`) a nie `$parameters` kde je stara hodnota.

Pridal som k tomu testy.